### DISCUSSION TITLE
How do I get SearXNG to show the full search URL

### DISCUSSION THREAD

**Discussion by RaptaG** [searxng/searxng#2612] [Q&A] [answered]
Instead of https://example.org/search
show https://example.org/search?q=test[other metadata info here]

I have already enabled "Query in the page's title" but it did nothing

**Comment by return42** [comment, marked as answer]
Your are looking for the HTTP GET `method` ..

```yaml
server:
  method: GET
```

the default is POST

https://github.com/searxng/searxng/blob/bcaaae699f01d44fa83dd8c251d9b30e5325d5e9/searx/settings.yml#L86-L88

**Comment by slrslr** [comment]
I have tried to add it as a feature request: https://github.com/searxng/searxng/issues/6228

If anyone faced this issue, here is how to workaround it: https://github.com/tiekoetter/searxng/issues/75#issuecomment-4659637880
